src/core/qgscoordinatetransform.cpp

```diff
@@ -652,7 +652,18 @@ void QgsCoordinateTransform::transformCoords( int numPoints, double *x, double *
   // if the source/destination projection is lat/long, convert the points to radians
   // prior to transforming
   ProjData projData = d->threadLocalProjData();
-#if PROJ_VERSION_MAJOR<6
+
+  int projResult = 0;
+#if PROJ_VERSION_MAJOR>=6
+  proj_errno_reset( projData );
+  proj_trans_generic( projData, direction == ForwardTransform ? PJ_FWD : PJ_INV,
+                      x, sizeof( double ), numPoints,
+                      y, sizeof( double ), numPoints,
+                      z, sizeof( double ), numPoints,
+                      nullptr, sizeof( double ), 0 );
+  projResult = proj_errno( projData );
+  // ewww - this logic is gross. We should drop support for PROJ 6.0 as quickly as possible and dump this code (in favour of built in methods used for >=6.1 builds)
```

Comment:
I believe you can get rid of the eww comment here.

Reply:
Thanks, fixed